src/base/utils/string.h

```diff
@@ -49,6 +47,14 @@ namespace Utils
         bool naturalCompareCaseInsensitive(const QString &left, const QString &right);
 
         QString wildcardToRegex(const QString &pattern);
+
+        template <typename T>
+        T unquote(const T &str)
+        {
+            if (str.startsWith('\"') && str.endsWith('\"'))
```

Comment:
Thank you for the clarification! I've tried to write a template function that takes quote chars as parameters, but I don't know how to make it handy and without an overhead :(. I don't know how to pass Qt char array in a simple way.